drivers/qwiic/qwiic.c

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2018 Jack Humbert <jack.humb@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "qwiic.h"
+
+void qwiic_init(void) {
```

Comment:
Before we get too many qwiic specific things does it make sense to break these into their own drivers? Otherwise we may end up in this weird situation in the future where devices that have both qwiic and non-qwiic applications will be confusing.